src/components/ProductTours/AdminOnboardingTours/flows/AdministerSubscriptionsFlow.tsx

```diff
@@ -72,40 +83,73 @@ const AdministerSubscriptionsFlow = ({
   const onBack = () => handleBackTour(ADMIN_TOUR_EVENT_NAMES.ADMINISTER_SUBSCRIPTIONS_BACK_EVENT_NAME);
 
   // Don't load the tour until we load the customer agreement
-  if (isLoadingCustomerAgreement) {
-    return [];
+  if (isLoadingCustomerAgreement || isLoadingAgreement) {
+    return [
+      {
+        target: `#${ADMINISTER_SUBSCRIPTIONS_TARGETS.SIDEBAR}`,
+        placement: 'right',
+        title: intl.formatMessage(messages.administerSubscriptionsTitle),
+        body: intl.formatMessage(messages.administerSubscriptionsStepOneBodySingleSub),
+        onAdvance,
+      }];
```

Comment:
nit: I would opt to return `[]` while entities needed to determine the flow steps are loading.  Otherwise, the user will see the tour contents change a split second after loading the entities (more noticeable when the user clicks the tour from a tab other than the subscriptions tab).

https://github.com/user-attachments/assets/8987028b-dac7-4a93-be7b-705dbc114d73

```suggestion
  if (isLoadingCustomerAgreement || isLoadingAgreement) {
    return [];
```